app/forms/personal_details_form.rb

```diff
@@ -0,0 +1,34 @@
+class PersonalDetailsForm
+  include ActiveModel::Model
+
+  attr_accessor :fullname, :email, :licence_id
+
+  def initialize(fullname: nil, email: nil, licence_id: nil)
+    @fullname = fullname
+    @email = email
+    @licence_id = licence_id
+  end
+
+  validates :fullname,
+    presence: {
+      message: "You must provide your full name"
+    }
+
+  validates :email,
+    presence: {
+      message: "You must provide your email address"
+    },
+    format: {
+      with: /.+@.+\..+/,
+      message: "Your email address must contain the @ symbol and end in a domain name (e.g. roger@example.com)"
```

Comment:
This would be "Enter an email address in the correct format, like name@example.com" 

https://design-system.service.gov.uk/patterns/email-addresses/#if-the-email-address-is-not-in-the-correct-format-and-there-is-no-example

Reply:
yes, thanks for that, I'll improve these here and fix up other messages in a PR to follow. Trello: https://trello.com/c/ZBmAX3XL/41-improve-error-messages-to-conform-to-govuk-guidance